From: Markus Blatt <markus@dr-blatt.de>
Date: Tue, 8 Nov 2016 21:38:41 +0100
Subject: libert.ecl-dev need to depend on ${misc:Depends}

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index fa062ed..d9c2962 100644
--- a/debian/control
+++ b/debian/control
@@ -12,7 +12,7 @@ Vcs-Browser: https://github.com/Ensembles/ert
 Package: libert.ecl-dev
 Section: libdevel
 Architecture: any
-Depends: libert.ecl1 (= ${binary:Version})
+Depends: ${misc:Depends}, libert.ecl1 (= ${binary:Version})
 Description: Ensemble based Reservoir Tool -- Development files
  ERT - Ensemble based Reservoir Tool is a tool for managing en ensemble
  of reservoir models.
